index.js

```diff
@@ -26,9 +26,10 @@ module.exports = function (opts) {
 		opts.inputSrc = file.contents;
 		opts.output = path.join(opts.dest, file.relative);
 		opts.outputHandler = function(filename, data, finished) {
+			filename = path.resolve(filename);
```

Comment:
This can be inline as it's not used anywhere else.

Not sure this is correct though. Shouldn't it be `path.resolve(file.base, filename)`?